### PR TITLE
i18n(fr): Add `guides/deploy/flyio.mdx`

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `guides/deploy/flyio.mdx` in French language

#### Related issues & labels (optional)

- Suggested label: i18n

🥳

### CODE REVIEW

File: src/content/docs/fr/guides/deploy/flyio.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    ```bash
+    fly launch
+    ```
+
+    `flyctl` détectera automatiquement Astro, configurera les paramètres corrects, construira votre image et la déploiera sur la plateforme Fly.io.
+
+## Générer votre fichier Astro Dockerfile
+
+Si vous n'avez pas encore de fichier Docker, `fly launch` va en générer un pour vous, ainsi que préparer un fichier `fly.toml`. Pour les projets `server` ou `hybrid`, ce fichier Docker inclura la commande de démarrage appropriée et les variables d'environnement.
```
Comment (pioupia) [inline src/content/docs/fr/guides/deploy/flyio.mdx:37]:
```suggestion
Si vous n'avez pas encore de Dockerfile, `fly launch` va en générer un pour vous, ainsi que préparer un fichier `fly.toml`. Pour les projets `server` ou `hybrid`, ce Dockerfile inclura la commande de démarrage appropriée et les variables d'environnement.
```

**Review by pioupia — CHANGES_REQUESTED** [review]

**Comment by thomasbnt** [comment]
Salut et merci pour ta review @pioupia ! 🚀

**Review by pioupia — APPROVED** [review]
LGTM

**Review by dreyfus92 — APPROVED** [review]
Merci les gars! LGTM ✅

**Pull request merged**